UnitsNet/CustomCode/Wrappers/Pressure.Wrapper.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnitsNet.Units;
+using UnitsNet.CustomCode.Units;
+
+namespace UnitsNet.CustomCode.Wrappers
+{
+    /// <summary>
+    /// From gratestas in #422
```

Comment:
`From gratestas in #422`
This does not belong in summary I think. At best it could be in `<remarks>`, but I think this is information that does not belong in source code. The pull request has this information and I think that is enough.